plugins/main/public/controllers/register-agent/utils/validations.tsx

```diff
@@ -42,16 +42,23 @@ export const validateAgentName = (value: any) => {
   if (value.length === 0) {
     return undefined;
   }
-  const regex = /^[A-Za-z.\-_,]+$/;
+  if (value.length < 2) {
+    return `The minimum length is 2 characters. ${validateCharacters(value)}`;
```

Comment:
**nitpick**: If the `validateCharacters` returns an empty string, the interpolated string will have a trailing space. Maybe we could avoid to adding this trailing space.

Reply:
Fixed in https://github.com/wazuh/wazuh-dashboard-plugins/pull/6105/commits/2be5b39994c074988e29b00c55bba76bc118d7b1